src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Rectangle extends Quadrangle {
+    public Rectangle(String color, int width, int height) {
+        super(color, width, height);
+    }
+
+    @Override
+    public double getArea() {
+        return super.getArea();
```

Comment:
Since `Rectangle` inherits `Quadrangle` and does not change the behavior of `getArea`, overriding it here is redundant. You can remove this method to use the implementation from the `Quadrangle` directly.